### ISSUE TITLE
Trucks trying to dump material at a location decide they can't and every other truck follows suit.

### ISSUE THREAD

**Issue by tkryton** [MaFi-Games/Captain-of-Industry-issues#659]
### Issue description

Initially I had four trucks declare that they were unable to dump waste material (dirt, rock, and limestone) on four tiles to create the beginnings of a spur. It eventually spiralled into fortyseven trucks that now can't deliver anything. I deleted the spur and still these trucks are now trying to access a nonexistent location.

This includes trucks that are already carrying previous cargo (construction parts, iron, et al). It's almost like they've all decided to go on strike because the four initial trucks couldn't do their jobs.

I've tried "recovering" some of the vehicles but they wind up stacked atop each other at the two depots.

The reason they couldn't get to their drop areas was because the land to the area was blocked by higher terrain that I inadvertently created and am now attempting to mine the hell out of it so I don't lose four or five days of failures and restarts.

### Expected behavior

I expected trucks loaded with other material, i.e. not waste, to finish dumping their material before trying to access a non-existent dumping zone.

### Reproduction steps

Initially created a dumping zone.
Trucks announced that they couldn't reach it.
Deleted unreachable dumping zone.
All trucks, even those assigned to routes, have joined a union and are refusing to work claiming that they "cannot reach destination."
Supervisor is considering quitting and moving or purchasing a nuke and turning the island into a glass parking lot.

### Game version

v0.4.8

### Attachments

[22-08-01_08-00-27_5083.log](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/9235606/22-08-01_08-00-27_5083.log)

**Comment by zuff-coi** [comment]
Would you be able to provide a save game of this situation? I could not reproduce it on my end with v0.4.9c

**Comment by tkryton** [comment]
I was playing with version 0.4.8. I got desperate and in an effort to save my progress in the game I ended up deleting all of my vehicles and rebuilding them at a couple of depots. So I’ve managed to recover the game. And, yes, I did manage to save the game prior to the mass culling.

Where things fell off the rails was just south of the default limestone mountain. Initially I built a “spur” that was roughly half way through the horizontal axis of said “mountain.” The spur was one tile wide and four tiles long pointing south(?) from the midpoint of the limestone mountain. Attached to this spur at its north end was a block roughly four by four tiles used to dump and build more land of trash (rock, dirt, slag, and limestone).

What I didn’t notice was that I forgot to change the setting on the dump zone from slope to flat and ended up creating a four tile wide dump site that blocked the spur. When two trucks started to complain that they couldn’t use that dump site, on the spur, I thought I’d just send the two excavators over and flatten out the area properly.

At this point the rest of the trucks joined the queue and complained that they couldn’t reach their destinations as well.

I’ve attached the saved version where it shows forty-seven pickups unable to complete their routing and a screen grab of the area where things went wrong.

Hope this helps,

Tom
